pkg/operator/ceph/csi/controller.go

```diff
@@ -299,3 +303,13 @@ func (r *ReconcileCSI) reconcile(request reconcile.Request) (reconcile.Result, e
 
 	return reconcileResult, nil
 }
+
+func setCSILogrotateParams(spec cephv1.ClusterSpec) {
+	logger.Debugf("set logrotate values in csi param")
+	CSIParam.CSILogRotation = spec.LogCollector.Enabled
```

Comment:
wont this cause problem if we have multiple ceph cluster and where one is Log enabled and for another its disabled, because we are going with assumption that we are always going to get the same cluster in first in list when we do List operation?

Reply:
Looped for the the list of cephclusters, 
Which ever is the first cephcluster in the list which has log rotation config, will use that

Reply:
> Looped for the list of cephclusters, Which ever is the first cephcluster in the list which has log rotation config, will use that

This makes sense. Anyway, this is expected to be temporary as the CSI operator should define these settings in its CRDs.